src/tempods/pages/__init__.py

```diff
@@ -1,21 +1,178 @@
+import numpy as np
 import solara
 from solara.alias import rv
+
+from tempods.components import SubsetControlWidget
 from ..layout import Layout
 
 
 @solara.component
 def Page():
-    with rv.Row():
-        with rv.Col(cols=12):
-            solara.Div("Tempo Data Story", classes=["display-1", "mb-8"])
-
-            solara.Div(
-                "Lorem ipsum dolor sit amet, consectetur adipiscing elit sed "
-                "do eiusmod tempor incididunt ut labore et dolore magna "
-                "aliqua. Ut enim ad minim veniam, quis nostrud exercitation "
-                "ullamco laboris nisi ut aliquip ex ea commodo consequat. "
-                "Duis aute irure dolor in reprehenderit in voluptate velit "
-                "esse cillum dolore eu fugiat nulla pariatur. Excepteur sint "
-                "occaecat cupidatat non proident, sunt in culpa qui officia "
-                "deserunt mollit anim id est laborum."
+
+    def _glue_setup() -> Tuple[JupyterApplication, IPyLeafletMapViewer, TimeSeriesViewer]:
+        app = jglue()
+
+        # TODO: We should probably change where the data lives
+        notebooks_dir = Path(__file__).parent.parent.parent.parent / "notebooks"
+        coastlines_path = notebooks_dir / "coastlines.geojson"
+        with open(str(coastlines_path), 'r') as f:
+            coastdata = json.load(f)
+        geo_json = GeoJSON(
+            data=coastdata,
+            style={
+                'color': 'black',
+                'opacity': 1,
+                'fillOpacity': 0,
+                'weight': 0.5
+            },
+        )
+
+        stadia_base_url = "https://tiles.stadiamaps.com/tiles/stamen_toner_lines/{z}/{x}/{y}{r}.png"
+        stadia_labels_url="https://tiles.stadiamaps.com/tiles/stamen_toner_labels/{z}/{x}/{y}{r}.png"
+
+        stadia_api_key = getenv("STADIA_API_KEY")
+        if stadia_api_key is not None:
+            stadia_base_url += f"?api_key={stadia_api_key}"
+            stadia_labels_url += f"?api_key={stadia_api_key}"
+        map_state = MapViewerState(basemap=TileLayer(url=stadia_base_url))
+
+        powerplant_datapath = notebooks_dir / "Power_Plants.csv"
+        powerplant_data: Data = app.load_data(str(powerplant_datapath))
+
+        asdc_url = "https://gis.earthdata.nasa.gov/image/rest/services/C2930763263-LARC_CLOUD/"
+        tempo_data = RemoteGeoData_ArcGISImageServer(asdc_url, name="TEMPO")
+        app.add_data(tempo_data)
+
+        # Our remote dataset does not have real components representing latitude and longitude. We link to the only components
+        # it does have so that we can display this on the same viewer without trigger and IncompatibleAttribute error
+        app.add_link(app.data_collection["Power_Plants"], 'Longitude',  app.data_collection["TEMPO"], 'Pixel Axis 0')
+        app.add_link(app.data_collection["Power_Plants"], 'Latitude', app.data_collection["TEMPO"], 'TEMPO_NO2_L3_V03_HOURLY_TROPOSPHERIC_VERTICAL_COLUMN_BETA')
```

Comment:
Also the variable name on the server changed, so this needs to be:

`TEMPO_NO2_L3_V03_HOURLY_TROPOSPHERIC_VERTICAL_COLUMN`

(i.e. just remove the _BETA)

Reply:
You'll also need the latest glue_map because I hacked the component name there instead of doing it properly. I've now hacked it to the current correct version and next time I'll certainly fix it properly!